src/as/math/MathUtils.ts

```diff
@@ -12,33 +12,41 @@
  * @see <a href="https://github.com/mrdoob/three.js/blob/master/src/math/Math">src/math/Math</a>
  */
 
-export const DEG2RAD: f32 = Mathf.PI / 180
-export const RAD2DEG: f32 = 180 / Mathf.PI
+const DEG2RAD: f32 = Mathf.PI / 180
+const RAD2DEG: f32 = 180 / Mathf.PI
 
-export function toString(source: i32, radix: i8): string {
-	let result: string = ''
+const toStringRadixSupported: bool = false // toString: The radix parameter is not supported yet.
 
-	return result
+/**
+ * Returns the respective basic value converted to a string.
+ */
+export function toString(number: i32, radix?: i8): string {
+	if (radix === null || radix > 1 && radix < 17) {
+		if (toStringRadixSupported) {
+			return number.toString(radix)
+		}
+		return _toString(number, radix)
+	}
+	throw new Error('number can be converted to string only on a radix between 2 and 36')
 }
```

Comment:
why this still have custom implementation?

Reply:
Because we shall keep it before a next code review. toString was already defined in `MathUtils` but not implemented.